package.json

```diff
@@ -14,6 +14,7 @@
     "synchronizer"
   ],
   "devDependencies": {
+    "@svgr/webpack": "^8.1.0",
```

Comment:
Removed in [491331d](https://github.com/MAYA-ZK/zk-dashboard/pull/11/commits/491331d59e33fb654a7b8b9ecaeedbc8910e8f94)